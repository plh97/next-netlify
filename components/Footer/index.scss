.Footer-container {
	width: 100%;
	box-shadow: 0px 2px 10px 2px rgba(0, 0, 0, 0.2);
	height: 30px;
	min-width: 1200px;
	display: flex;
	align-items: center;
	justify-content: center;
	font-size: 12px;
	// box-shadow: var(--box-shadow-base);
	z-index: 1;
	flex-shrink: 0;

	span {
		margin-right: 2px;
		margin-left: 2px;
	}
}
