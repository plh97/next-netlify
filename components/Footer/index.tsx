import React, { Component } from 'react'
import './index.scss'

export default function Footer () {
	return (
		<div className="Footer-container">
			Â©<span>Copyright</span>
			<span>2017-2023</span>
			<span>
				<b>plh-blog.netlify.app</b>
			</span>
		</div>
	)
}
