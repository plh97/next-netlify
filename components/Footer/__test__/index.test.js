// Link.react.test.js
import React from 'react'
import Footer from '../index.jsx'
import renderer from 'react-test-renderer'

it('renders correctly', () => {
	const tree = renderer.create(<Footer />).toJSON()
	expect(tree).toMatchSnapshot()
})
