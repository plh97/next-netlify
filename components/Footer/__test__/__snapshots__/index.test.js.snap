// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div
  className="Footer-container"
>
  Â©
  <span>
    Copyright
  </span>
  <span>
    2017-2019
  </span>
  <span>
    <b>
      pipk.top
    </b>
  </span>
</div>
`;
